### PR TITLE
Tweak `--upgrade` command to also replace `@liveblocks/react-{comments => ui}`

### PR DESCRIPTION
**Pull request by nvie**
…and also warn the user if a dependency on `@liveblocks/core` is detected.

https://github.com/liveblocks/liveblocks/assets/83844/5cfbe632-fa9a-43d7-8cd9-6ac52606a296

### CODE REVIEW

File: packages/create-liveblocks-app/bin/templates/upgrade/upgrade-template.ts [hunk 1]
```diff
@@ -25,43 +25,86 @@ export async function create() {
...
+  const depsToUpgrade = liveblocksDeps.map((d) => {
+    if (d === "@liveblocks/react-comments") {
+      return "@liveblocks/react-ui";
+    } else {
+      return d;
+    }
+  });
+
+  const depsToUninstall = liveblocksDeps.filter(
+    (d) => d === "@liveblocks/react-comments"
```
Comment (nvie) [inline packages/create-liveblocks-app/bin/templates/upgrade/upgrade-template.ts:50]: Hardcoded for now, we can make it configurable later.

File: packages/create-liveblocks-app/bin/templates/upgrade/upgrade-template.ts [hunk 2]
```diff
@@ -25,43 +25,86 @@ export async function create() {
...
+        `Please note that @liveblocks/core contains private APIs only.`
+      )
+    );
+    console.log(
+      c.bold.yellowBright(
+        `It is recommended to only rely on @liveblocks/client.`
+      )
+    );
+    console.log();
+  }
```
Comment (nvie) [inline packages/create-liveblocks-app/bin/templates/upgrade/upgrade-template.ts:109]: @marcbouchenoire Do you think this is the right coloring style?

**Review by marcbouchenoire — APPROVED** [review]
Nice! Feel free to remove the `package.json` part of the `react-comments-to-react-ui` codemod now that it's handled here.

File: packages/create-liveblocks-app/bin/templates/upgrade/upgrade-template.ts (hunk 2 above)
Comment (marcbouchenoire) [reply packages/create-liveblocks-app/bin/templates/upgrade/upgrade-template.ts:109]: Not 100% sure but it should be alright 👍

**Comment by nvie** [comment]
> Nice! Feel free to remove the `package.json` part of the `react-comments-to-react-ui` codemod now that it's handled here.

@marcbouchenoire Like this? d98d43a740c6c043358f45b9a8555b2735e87367

**Pull request merged**